subprojects/docs/src/docs/release/notes.md

```diff
@@ -86,6 +86,15 @@ TODO - Java lambdas are supported, and unsupported captured values are reported.
 TODO - File collections queried at configuration time are treated as configuration inputs.
 TODO - File system repositories are fully supported including dynamic versions in Maven, Maven local, and Ivy repositories
 
+## Tooling API improvements
+
+- [Build launched via TAPI ignores logger level settings of target build](https://github.com/gradle/gradle/issues/19340)
+
+  `org.gradle.logging.loglevel` settings in `gradle.properties` are now recognized when a build is executed with the Tooling API.
+  This was a problem caused executions triggered from within the IDE to ignore loglevel settings in `gradle.properties`. 
+  A workaround had to be implemented. This workaround can be avoid from 8.1 onwards.  
```

Comment:
Here is the recommended format for each section: 
- Set the context - what are we even talking about (don’t assume the user already knows)
- Express the problem or use case in the given context
- Explain how the new release addresses that problem or use case
- Provide build script snippet or screenshot illustrating the new feature, if applicable
- Link to the full documentation for more details

Reply:
I hope this comes closer to the recommendation now.
I think a screenshot is not helpful here, at least I can't think of one.